content/content-strategy/define-user-content-needs/create-actionable-artefacts/top-tasks/diagram.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: blocks/image_content_block
+image: content-strategy/top-tasks.svg
+link: content-strategy/top-tasks.svg
+reverse: false
+stackPosition: top
+imageAlt: top user tasks
+level: 2
+display: 3
+long_description: new-long-description.md
+---
+
+### [3]Step 4: Get users to vote
```

Comment:
This section heading should be added to the YAML at the top of the file as `title: Step 4: Get users to vote`
The`level` and `display` params in the YAML need to be adjusted to '3' and '2' respectively.

Reply:
Steve fixed.